### PR TITLE
[Fix] fix "exports" for node 13.0-13.6 take 2

### PR DESCRIPTION
**Pull request by ljharb**
Followup to #41.

Sorry about the churn!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,14 +35,17 @@
...
+  "exports": {
+    ".": [
+      {
+        "import": "./src/index.mjs",
+        "require": "./dist/index.cjs",
+        "default": "./dist/index.cjs"
+      },
+      "./dist/index.cjs"
+    ],
+    "./package.json": "./package.json"
```
Comment (ljharb) [inline package.json:47]: this ensures tooling can access package.json

File: package.json [hunk 2]
```diff
@@ -35,14 +35,17 @@
...
-  "exports": [
-    {
-      "import": "./src/index.mjs",
-      "require": "./dist/index.cjs",
-      "default": "./dist/index.cjs"
-    },
-    "./dist/index.cjs"
-  ],
+  "exports": {
+    ".": [
```
Comment (ljharb) [inline package.json:39]: the dot means "main" here

**Comment by twada** [comment]
Thanks!

**Comment by twada** [comment]
unassert 2.0.2 is out. Thanks again!
https://github.com/unassert-js/unassert/blob/master/CHANGELOG.md#202-2023-04-20

**Comment by ljharb** [comment]
Thank you!!

File: package.json (hunk 2 above)
Comment (twada) [reply package.json:39]: Can node 13.0-13.6 recognize "." as well?

Comment (ljharb) [reply package.json:39]: ahhh good point. node 13.2 - 13.6 can, they just will fall back to the string format - but i think node 13.0 and 13.1 won't, and will still require another array fallback?

I'll make a third PR if that's indeed the case, but this is still an improvement as is :-)

**Pull request merged**